plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1492,3 +1492,14 @@ func (c *VolumeBindingChecker) predicate(pod *v1.Pod, meta algorithm.PredicateMe
 	glog.V(5).Info("All PVCs found matches for pod %v/%v, node %q", pod.Namespace, pod.Name, node.Name)
 	return true, nil, nil
 }
+
+type FakePersistentVolumeClaimInfo []v1.PersistentVolumeClaim
```

Comment:
Why is this block moved to this file? It does not belong here.

Reply:
This will be used by equivalence_cache_test.go, maybe create a test helper pkg?